docs/quickstart.rst

```diff
@@ -27,11 +27,11 @@ Before using Chassis, this is how your system should be set up:
 
   * **OS X**: You already have Bonjour available.
 
-  * **Windows**: If you have iTunes, Safari, Bonjour Print Services, or Creative
+  * **Windows**: If you have Safari, Bonjour Print Services, or Creative
```

Comment:
Safari is actually also no longer available, and Creative Suite 3 should be Creative Cloud, while we're here ;)

Reply:
Done!